packages/wonder-blocks-popover/src/components/popover-content.tsx

```diff
@@ -177,8 +182,11 @@ export default class PopoverContent extends React.Component<Props> {
                 {typeof icon !== "string" ? (
                     icon
                 ) : (
-                    // eslint-disable-next-line jsx-a11y/alt-text -- TODO: Address a11y error
-                    <StyledImg src={icon} style={styles.icon} />
+                    <StyledImg
+                        src={icon}
+                        style={styles.icon}
+                        alt={iconAlt || ""}
```

Comment:
We expose a prop to set the icon alt text or set to `""` for decorative purposes